### ISSUE TITLE
Remove the DPG activity column from the registry

### ISSUE THREAD

**Issue by prajectory** [DPGAlliance/publicgoods-website#19]
The activity column is not updating in tandem with the reality. Is it possible to mask this column for now completely? Not have it on the registry as a part of incremental changes?

**Comment by nathanfletcher** [comment]
@prajectory I just finished doing a deep dive on this issue. 

The activity column of the registry is based on the GitHub repository URL of the project submitted.
The graphs generated are accurate. However, I noticed that projects like African Storybook have two repositories.
The repository African Storybook submitted is this link https://github.com/global-asp/global-asp
They also have another newer repository similar to the first one https://github.com/global-asp/global-asp
In such a case, both activities cannot be used, and if they want the recent repos's graph to be captured on the DPG registry, they would need to submit an update to you through their account on the web app.

Aside from cases like that, all other software projects on the registry have accurate activity graphs. As such, it would be too drastic to mask that column completely.
I would suggest that we identify DPGs whose activity column is not updating in tandem with the reality, and let them provide to the registry, the real GitHub repo whose activity they would want to show.
